riot/lol/constants.go

```diff
@@ -17,12 +17,10 @@ const (
 	endpointGetLeague                    = endpointLeagueBase + "/leagues/%s"
 	endpointStatusBase                   = endpointBase + "/status/v3"
 	endpointGetStatus                    = endpointStatusBase + "/shard-data"
-	endpointMatchBase                    = endpointBase + "/match/v4"
-	endpointGetMatch                     = endpointMatchBase + "/matches/%d"
-	endpointGetMatchesByAccount          = endpointMatchBase + "/matchlists/by-account/%s?beginIndex=%d&endIndex=%d"
-	endpointGetMatchTimeline             = endpointMatchBase + "/timelines/by-match/%d"
-	endpointGetMatchIDsByTournamentCode  = endpointMatchBase + "/matches/by-tournament-code/%s/ids"
-	endpointGetMatchForTournament        = endpointMatchBase + "/matches/%d/by-tournament-code/%s"
+	endpointMatchBase                    = endpointBase + "/match/v5"
+	endpointGetMatchIDs                  = endpointMatchBase + "/matches/by-puuid/%s/ids?start=%d&count=%d"
+	endpointGetMatch                     = endpointMatchBase + "/matches/%s"
+	endpointGetMatchTimeline             = endpointMatchBase + "/matches/%s/timeline"
```

Comment:
what about the endpoints that do not have an implementation in the new version? IMO, we should keep them available until they have been deprecated.

Reply:
> match-v4 is being deprecated on Monday, August 23rd, 2021
I figured i'd remove them, given that this PR wwould be ready to merge when there's < 1 month left before the old endpoints go away. People wanting to continue to use v4 endpoints can not update or use a specific commit (or there could be a branch made for v5 support wwhich gets merged once v4 is deprecated)